packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -260,425 +309,456 @@ export interface ComboBoxProps
   /**
    * Provide the text that is displayed when the control is in warning state
    */
-  warnText?: ReactNodeLike;
+  warnText?: ReactNode;
```

Comment:
Here we used to have `ReactNodeLike` but maybe we could change to `string`, since we are using only text.

Reply:
This is also a `node` down in the proptypes.